src/solvers/dgsem_tree/indicators_1d.jl

```diff
@@ -201,6 +201,40 @@ function (indicator_max::IndicatorMax)(u::AbstractArray{<:Any,3},
   return alpha
 end
 
+function create_cache(::Type{IndicatorClamp}, equations::AbstractEquations{1}, basis::LobattoLegendreBasis)
+
+  alpha = Vector{real(basis)}()
+
+  return (; alpha, indicator_threaded, basis.weights)
+end
```

Comment:
Where is `indicator_threaded` defined - and is it required?

Reply:
Removed.